docs/models.yml

```diff
@@ -1044,11 +1057,7 @@ agenda_item:
     equal_fields: meeting_id
   projection_ids:
     type: relation-list
-    to: projection/element_id
-    equal_fields: meeting_id
-  current_projector_ids:
-    type: relation-list
-    to: projector/current_element_ids
+    to: projection/content_object_id
```

Comment:
Why did you remove the `equal_fields`? If this was by mistake, please add back (same for all other places below).

Reply:
It is in the line below your comment.

Reply:
Oops, my bad